registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleManager.java

```diff
@@ -291,4 +303,8 @@ private boolean isMessageExpired(MessageDTO messageDTO, long messageExpiryTimeLi
 		}
 	}
 
+	protected String getPropertyPrefix() {
```

Comment:
Since there are non-stage implementations of MosipVerticleManager such as ConsumerVerticle, there needs a default implementation to return empty string in getPropertyPrefix method. Also moving to EventBusManager interface requires the method to be public which is not needed. So no change is applied here.

Reply:
Making the getPropertyPrefix method to abstract to ensure all extension classes of MosipVerticleManager to implement that method. Not moving that method to EventBusManager since it is not an API to expose publicly.